### PR TITLE
Boilerplate/config

### PR DESCRIPTION
**Pull request by alina-kho**
# Description

Our class has faced a issue while setting up the boilerplate and database for the final project using docker. This was due to a couple of small bugs in the env file. 

Thus, this PR contains fixes of env.example for the future classes:

* Database credentials are edited in accordance with MySQL vars
* MYSQL_PORT is added as a config var

# How to test?

Try setting up the boilerplate and compiling database using docker

### CODE REVIEW

**Review by te-online — APPROVED** [review]

File: packages/server/.env.example [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Database credentials
```
Comment (miltonhultgren) [inline packages/server/.env.example:1]: @dpfernandes If we enable Actions again, we'll have to make sure the workflow plays well with this env file (that's why I changed this file for FP21, but maybe the workflow should be changed instead?)

**Comment by miltonhultgren** [comment]
@alina-kho Is anything happening in this PR or should we close it?

**Comment by te-online** [comment]
What if we change the variables in way that will enable Actions to continue to work and solve the local problem at the same time? 🤓

**Comment by CK-HYF-DK** [comment]
@dpfernandes not sure how to move forward with this one

**Pull request closed without merging**